state-chain/pallets/cf-ingress-egress/src/mock_eth.rs

```diff
@@ -110,6 +112,44 @@ impl NetworkEnvironmentProvider for MockNetworkEnvironmentProvider {
 
 impl_mock_runtime_safe_mode! { ingress_egress_ethereum: PalletSafeMode<()> }
 
+thread_local! {
+	pub static USE_LIMITS: RefCell<bool> = RefCell::new(false);
+}
+
+pub struct MockFetchesTransfersLimitProvider;
```

Comment:
Why define this here instead of in the trait crate, same as all the other mocks?

Reply:
Because it's only used in this pallet at the moment and also only used by this pallet.

Reply:
Ok but the convention we (try to) follow is to define the mocks where the trait is defined. This works well and I don't see a good reason to break with that convention here.